src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/twoFactorRequired.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import {capitalize} from 'lodash';
+import Alert from 'app/components/alert';
+import AsyncComponent from 'app/components/asyncComponent';
+import space from 'app/styles/space';
+
+let StyledAlert = styled(Alert)`
+  margin: ${space(3)} 0;
+`;
+
+class TwoFactorRequired extends AsyncComponent {
+  getEndpoints() {
+    return [['organizations', '/organizations/']];
+  }
+
+  renderBody() {
+    let orgsRequire2FA = this.state.organizations
+      .filter(org => org.require2FA === true)
+      .map(({name}) => {
+        return capitalize(name);
+      });
+    let multipleOrgs = orgsRequire2FA.length > 1;
+    let formattedNames = orgsRequire2FA.join(', ').replace(/,(?!.*,)/gim, ' and');
+
+    if (!orgsRequire2FA.length) {
+      return null;
+    }
+
+    return (
+      <div>
+        {multipleOrgs ? (
```

Comment:
We don't need this ternary here since the components are essentially the same. I would extract the `organization` string into a variable so that we can simplify this.